.pre-commit-config.yaml

```diff
@@ -218,39 +218,26 @@ repos:
     - tests/
     pass_filenames: false
 
-- repo: local
+- repo: https://github.com/PyCQA/pylint
+  rev: v2.12.2
```

Comment:
This suddenly jumps a few minor versions from `~= 2.8.0` set up in the tox env now. I guess it would have to be a separate PR demonstrating that this bump is okay and does not create problems on its own.

Reply:
I've been using the PR to find and fix anything that comes up.... If I understand this, this PR can't merge until we know this bump is ok and doesn't create problems on it's own right?

Reply:
Yeah, and also I'm not sure if stopping to reuse the requirements sources creates more problems than it solves.